src/Microsoft.Health.Fhir.Core/Features/Validation/Narratives/NarrativeHtmlSanitizer.cs

```diff
@@ -289,14 +283,6 @@ private static void ValidateAttributes(IElement element, Action<IElement, IAttr>
                 {
                     onInvalidAttr(element, attr);
                 }
-
-                if (string.Equals("src", attr.Name, StringComparison.OrdinalIgnoreCase))
-                {
-                    if (!Src.Any(x => attr.Value.StartsWith(x, StringComparison.OrdinalIgnoreCase)))
-                    {
-                        onInvalidAttr(element, attr);
-                    }
-                }
```

Comment:
Can we add a few examples to the tests of what this now supports? (i.e. the IG image example) Are there any negative test cases we should add?

Reply:
I don't think there are any negatives to add. @michael-wilson-au added a negative test as part of the first PR, but my change only removes a restriction. I'll add some examples of what is supported.